zoo_frontend/src/pages/kitchen/prep/kitchen.jsx

```diff
@@ -9,172 +11,151 @@ import NativeSelect from '@material-ui/core/NativeSelect';
 import NextIcon from '@material-ui/icons/NavigateNext';
 import PrevIcon from '@material-ui/icons/NavigateBefore';
 import Paper from '@material-ui/core/Paper';
-import KitchenView from '../../../components/KitchenView';
+import KitchenView from './KitchenView';
 
-// import { hasAccess, Home, Diet } from '../PageAccess';
+import { Kitchen } from '../../PageAccess';
 
 import {
-  DeliveryContainers,
   DietChanges,
   Diets,
   FoodPrepTables,
   PrepNotes,
-  Species,
 } from '../../../api';
 
+
 export default class extends Component {
   static propTypes = {
     // account: PropTypes.object.isRequired,
-    token: PropTypes.string,
+    // token: PropTypes.string,
     classes: PropTypes.object.isRequired,
-    FoodPrepTables: PropTypes.array.isRequired,
+    prepTables: PropTypes.array.isRequired,
     date: PropTypes.string.isRequired,
-    // eslint-disable-next-line react/no-unused-prop-types
-    PrepDiets: PropTypes.array.isRequired,
-    PrepDietsSub: PropTypes.array.isRequired,
+    prepDiets: PropTypes.array.isRequired,
+    prepDietsSub: PropTypes.array.isRequired,
+    prepNotes: PropTypes.array.isRequired,
+    dietChanges: PropTypes.array.isRequired,
+    table: PropTypes.string,
+    diet: PropTypes.string,
   };
 
   static defaultProps = {
-    token: '',
+    // token: '',
+    table: 0,
+    diet: 0,
   };
 
   static async getInitialProps({ query, authToken }) {
-    const serverFoodPrepTablesAPI = new FoodPrepTables(authToken);
-    const serverDietsAPI = new Diets(authToken);
+    const prepTablesAPI = new FoodPrepTables(authToken);
+    const dietsAPI = new Diets(authToken);
+    const dietChangesAPI = new DietChanges(authToken);
+    const prepNotesAPI = new PrepNotes(authToken);
 
     try {
-      const [AllFoodPrepTables, AllFoodPrep] = await Promise.all([
-        serverFoodPrepTablesAPI.getFoodPrepTables(),
-        serverDietsAPI.getAnimalPrep(query.date), // this.props.date !!!!!!!!!!
+      const [prepTables, foodPrep, dietChanges, prepNotes] = await Promise.all([
+        prepTablesAPI.getFoodPrepTables(),
+        dietsAPI.getAnimalPrep(query.date), // this.props.date !!!!!!!!!!
```

Comment:
is your comment just saying it's the equivalent of this.props but from the URL? loving the !!'s lol. just want to make sure you want to keep this note ;)